suse/rdma-core.spec

```diff
@@ -67,6 +68,7 @@ BuildRequires:  pkgconfig(systemd)
 Requires:       dracut
 Requires:       kmod
 Requires:       systemd
+Requires:       udev
 
```

Comment:
Isn't this implied by requires: systemd?

Reply:
It probably is, but at least it's explicit (and avoid some warning in the build system)